gms/gossiper.hh

```diff
@@ -225,6 +225,9 @@ private:
     std::unordered_map<inet_address, clk::time_point> _shadow_unreachable_endpoints;
     utils::chunked_vector<inet_address> _shadow_live_endpoints;
 
+    // duplicate live endpoints across all other shards.
```

Comment:
s/duplicate/replicate (standard terminology used elsewhere)
please mention that this can only be called from shard 0